src/addons/transitions/ReactCSSTransitionGroup.js

```diff
@@ -57,9 +57,16 @@ var ReactCSSTransitionGroup = React.createClass({
   },
 
   render: function() {
+    var {
+      transitionName,
+      transitionAppear,
+      transitionEnter,
+      transitionLeave,
+      ...transferredProps
+    } = this.props;
```

Comment:
Do we care?  Is someone fixing the transpiler soon, or do you want me to use Object.assign and manually remove the object properties?